maven_install.json

```diff
@@ -1,6 +1,6 @@
 {
     "dependency_tree": {
-        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": 607867898,
+        "__AUTOGENERATED_FILE_DO_NOT_MODIFY_THIS_FILE_MANUALLY": -512671146,
         "conflict_resolution": {},
```

Comment:
A question outside the scope of this PR.  It looks like this file is auto generated from the maven rule.  Should this be checked into SCM?

Reply:
I think so. So you can use the external_jvm_rules feature without this file, but anytime someone builds it will resolve the transitive dependencies and can lead to issues. Also performance and ability to do offline builds are impacted. Some good info on the feature here: https://github.com/bazelbuild/rules_jvm_external#pinning-artifacts-and-integration-with-bazels-downloader

One way to think of it is this. Before the WORKSPACE file had a longer list of Jar artifacts with SHA256, the tool now produces this file with the various files.

One key thing that I worry might trip someone up is that changes to the list of Maven dependencies in the WORKSPACE file aren't automatically honored. You have to first run `bazel run @unpinned_maven//:pin` to update the file. I put a comment in the WORKSPACE file as a reminder in case anyone tried to update versions of dependencies.

Reply:
Ok.  Thanks for the info.  I'll make sure to read up on this.